### DISCUSSION TITLE
Unable to generate graph.ph file.

### DISCUSSION THREAD

**Discussion by mayank45511** [deepmodeling/deepmd-kit#1481] [Q&A] [answered]
Hi,
I am trying to train my datasets. I am not getting any errors while training but the output database called graph.pb is not generating.
Can anybody please tell me if it is any kind of bug or I have to modify scripts?

Thank You.

**Comment by njzjz** [comment, marked as answer]
Did you execute `dp freeze`?

**Reply by mayank45511** [reply]
Yes, I had run dp freeze but was getting the same problem. After reinstalling the DeepMD-kit, issue has been solved.

**Comment by ooooppps** [comment]
dp freeze -o graph
